### PR TITLE
Problem: not using latest play-cpp-sdk (fix #68)

### LINKED ISSUE #68: Problem: not using latest play-cpp-sdk
update latest version

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: updated play-cpp-sdk for windows callback issue

### CODE REVIEW

**Comment by leejw51crypto** [comment]
updating now

**Comment by leejw51crypto** [comment]
updated

File: CronosPlaySdk/Plugins/CronosPlayUnreal/CronosPlayUnreal.uplugin [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
 	"FileVersion": 3,
-	"EngineVersion": 4.27.2,
```
Comment (tomtau) [inline CronosPlaySdk/Plugins/CronosPlayUnreal/CronosPlayUnreal.uplugin:3]: Not sure why this is removed? I saw previously @rarcif added it https://github.com/cronos-labs/play-unreal-plugin/pull/66 but there was no issue linked, so I'm not sure what the problem was

**Review by tomtau — APPROVED** [review]
how are those binaries on Git LFS generated? Could the Unreal's build script download them instead of putting it on Git?

File: CronosPlaySdk/Plugins/CronosPlayUnreal/CronosPlayUnreal.uplugin (hunk 1 above)
Comment (leejw51crypto) [reply CronosPlaySdk/Plugins/CronosPlayUnreal/CronosPlayUnreal.uplugin:3]: with EngineVersion, i cannot open the project.
i'll find another solution.

Comment (leejw51crypto) [reply CronosPlaySdk/Plugins/CronosPlayUnreal/CronosPlayUnreal.uplugin:3]: restored EngineVersion, confirmed working in mac

Comment (leejw51crypto) [reply CronosPlaySdk/Plugins/CronosPlayUnreal/CronosPlayUnreal.uplugin:3]: changing to "4.27.2" , working
4.27.2 is not json number, cannot be read as valid json value

**Review by damoncro — APPROVED** [review]

File: CronosPlaySdk/Plugins/CronosPlayUnreal/CronosPlayUnreal.uplugin (hunk 1 above)
Comment (leejw51crypto) [reply CronosPlaySdk/Plugins/CronosPlayUnreal/CronosPlayUnreal.uplugin:3]: log location:
~/Library/Logs/Unreal Engine/CronosPlaySdkEditor/CronosPlaySdk.log

**Comment by leejw51crypto** [comment]
seems like dylib issue
run copydll.sh beforehand, otherwise ue4 editor will not show up, because plugin phase is `predefault`

**Pull request merged**